### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by rushemaa**
### Hello Microverse
 This project has the following files:

 - An index HTML
 - External CSS 
 - RAEDME and 
 - .gitignore

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Hollo microverse</title>
+  </head>
+  <body>
+    <h1 class="header">Hello Microverse!</h1>
+  </body>
+</html>
```
Comment (iambenkis) [inline index.html:13]: - [ ] Good job @rushemaa , adding this page however as required you should install linters in your projects. They will allow you to eliminate some common problem in your code and make it looks professional :+1: 
Kindly follow this [link ](https://github.com/microverseinc/linters-config/tree/master/html-css) and install them :100:

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @rushemaa 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne:  
- Good commit history :heavy_check_mark:   
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Good job creating a PR for your project :+1: , however , it could be better if you add a PR summary to make known to the CR what you make in your branch :+1: , Kindly add it here

Screenshot from 2023-02-14 17-56-07


- [ ] Every project you will make at microverse or in a future job , documentation is a important part of project development. It could be better if you add a README file that stick to the [README template](https://github.com/microverseinc/readme-template) given by microverse, and you should customize it according to the project 

Screenshot from 2023-02-14 17-58-38

- [ ] Kindly make sure your project has  a .gitignore file listing test.md file and node_modules directory (as it can be added by accident while setting up Linters).

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shella12 — COMMENTED** [review]
Hi @rushemaa , 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project with a PR link to a Merged file please submit a valid link (open PR, not merged or closed) for the review

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by rushemaa** [comment]
@shella12  This PR is merged, and I am unable to return it to the original status, I tried to revert the merge and now I have 2 PR. I can't even open the first one

**Pull request merged**